### DISCUSSION TITLE
Tutorial

### DISCUSSION THREAD

**Discussion by monafaraji** [NREL/MATBOX_Microstructure_analysis_toolbox#13] [Q&A]
Hi and congrats for the great work you have done and also making it open-source! 
May I ask if you have a tutorial available for this toolbox? a video or a recording of a presentation just to start with a simple example?

Many thanks
Mona

**Comment by fussegli** [comment]
Hi Mona,
Thank you very much to start a discussion!
I plan to make such tutorial videos, and since you are the first to ask for one, which module would you like to have one first?
Best,
Francois.

**Reply by monafaraji** [reply]
Thanks @fussegli for replying, 
that's great, 
For me it would be from the basics, Microstructure generation and then Filtering, 
Can't wait to hear about that and keep going with the great job!
Best,
Mona

**Reply by fussegli** [reply]
Hi Mona, I just updated the microstructure generation documentation, you can have a look to see how to generate a complex microstructure from A to Z.
For video tutorial, I am waiting for the journal publication, which should be any time soon now.
Best,
Francois
